samples/nrf9160/sms/src/main.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```

Comment:
SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
(Old BSD-5 is present in several files)